mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
+python_version = 3.7
```

Comment:
I'm guessing this makes sense because 3.7 is our oldest supported version?

Reply:
Yep this indicates the minimum python version supported